metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/data_resolver.py

```diff
@@ -472,7 +472,12 @@ def _get_pages_by_report(self, workspace: Workspace, report_id: str) -> List[Pag
         )
 
         # Check if we got response from PowerBi
-        response.raise_for_status()
+        if response.status_code != 200:
```

Comment:
Let's add a common function to log debug messages if status_code is not equal to 200